homeassistant/components/zha/core/typing.py

```diff
@@ -10,33 +10,8 @@
 
 # pylint: disable=invalid-name
 CALLABLE_T = TypeVar("CALLABLE_T", bound=Callable)
-ChannelType = "ZigbeeChannel"
-ChannelsType = "Channels"
-ChannelPoolType = "ChannelPool"
-ClientChannelType = "ClientChannel"
-ZDOChannelType = "ZDOChannel"
-ZhaDeviceType = "ZHADevice"
-ZhaEntityType = "ZHAEntity"
-ZhaGatewayType = "ZHAGateway"
-ZhaGroupType = "ZHAGroupType"
 ZigpyClusterType = zigpy.zcl.Cluster
 ZigpyDeviceType = zigpy.device.Device
 ZigpyEndpointType = zigpy.endpoint.Endpoint
 ZigpyGroupType = zigpy.group.Group
 ZigpyZdoType = zigpy.zdo.ZDO
```

Comment:
I did start to replace these too. IMO there isn't really a need for these aliases.
Maybe that's a task for a followup PR?

Reply:
I agree, for a follow-up PR as this one is big enough IMO